src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -149,7 +178,7 @@ ucs_status_t uct_ib_mlx5_get_cq(struct ibv_cq *cq, uct_ib_mlx5_cq_t *mlx5_cq)
     mlx5_cq->uar       = uct_dv_get_info_uar0(dcq.dv.uar);
 #endif
     mlx5_cq->dbrec     = dcq.dv.dbrec;
-    memset(&mlx5_cq->cq_decomp, 0, sizeof(uct_ib_mlx5_cq_decomp_t));
+    memset(&mlx5_cq->cq_decompress, 0, sizeof(uct_ib_mlx5_cq_decompress_t));
```

Comment:
My previous comment is wrong. We need to initialize it because we still using `cq_decomp.left` to indicate that decompression is in progress. We can avoid it by hacking the `op_own` field somehow, but I think that this approach looks more straightforward and simple.

WDYT?

Reply:
given yesterday discussion in which place do we use decomp.left before decompression started the first time?

Reply:
It's [here](https://github.com/openucx/ucx/pull/7992/files#diff-9f51a455ceb0af94ca68ed6bcc36f7cec028c7fdaa2eb1af47da3fb5074bf949R201).

It seems like we still need some way to detect first and following CQEs in compression block.

Reply:
that link points to "memcpy(&cq_decompress->title, title, sizeof(struct mlx5_cqe64));" where we seem to overwrite the title

Reply:
Oh, that's strange, when I check it last time, it points to the following function:
```c
static UCS_F_ALWAYS_INLINE struct mlx5_cqe64*
uct_ib_mlx5_iface_cqe_decompress(struct mlx5_cqe64 *cqe,
                                 uct_ib_mlx5_cq_t *cq)
{
    if (cq->cq_decompress.left > 0) {
        return uct_ib_mlx5_iface_cqe_decompress_continue(cq);
    }
    return uct_ib_mlx5_iface_cqe_decompress_start(cqe, cq);
}
```

Reply:
1. couldn't we unite "cq_decompress.left" and "cq_decompress.mini_idx"?
2. i guess it would be enough to initialize cq->cq_decompress.left?
 we can still memset everything, but just comment what is the field we wanted to make sure is initialized

Reply:
1. Yes we can, as we discussed earlier.
2. Yes, it would be enough. Will add the comment.